src/tests/components/ForecastSummary.test.js

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import ForecastSummary from "../../components/ForecastSummary";
+
+describe("ForecastSummary", () => {
+  const validProps = {
+    date: "Thu Jan 01 1970",
+    description: "Stub description",
+    icon: "800",
+    temperature: {
+      min: 12,
+      max: 22,
+    },
+    onSelect: () => {},
```

Comment:
If you have function that you are passing as a prop, I suggest using `jest.fn()` instead of `() => {}`. Then, if you add tests for user interactions (e.g. a button click) you can test if the button has been clicked by checking if the `jest.fn()` has been called. You can read more about testing fire events [here](https://testing-library.com/docs/dom-testing-library/api-events/)